engine/src/main/java/com/redhat/cloud/notifications/recipients/rbac/RbacServiceToService.java

```diff
@@ -23,6 +23,7 @@ public interface RbacServiceToService {
     @Produces(MediaType.APPLICATION_JSON)
     RbacGroup getGroup(
             @HeaderParam("x-rh-rbac-account") String accountId,
+            @HeaderParam("x-rh-rbac-org-id") String orgId,
```

Comment:
Is `x-rh-rbac-org-id` a standard header used accross many products? I'm asking for my current work on policies.

Reply:
I did not check their code, but that's how I understood Lindani. I would say yes

Reply:
IIRC that is the standard way of sending the org-id. I guess that's how we are going to get the org-id from 3scale- unsure what's turnpike going to do.

Reply:
Hopping in late here, but this is how service-to-service requests are validated in RBAC. There are only a handful of services currently talking directly to RBAC. When RBAC changes tenancy to use org_id, we'll require this header in order to direct the request to the correct tenant.

Reply:
Thanks @coderbydesign !